Add support for custom builds (#224)
simple-icons/simple-icons-font: README.md

In this example we use the `<i>` tag, but any inline HTML tag should work as you expect.

## Custom Builds

> You can specify which icons need to be build for a better file size.

Reviewer: ```suggestion
You can specify which icons need to be build for a smaller file size.
```

1. Clone and install dependencies:

```shell
git clone git@github.com:simple-icons/simple-icons-font.git
cd simple-icons-font
npm install
```

2. Use environment variable `SI_FONT_SLUGS_FILTER` to filter icons to include:

```shell
SI_FONT_SLUGS_FILTER=github,simpleicons npm run build
```

> Here are some available environment variables for custom build:

Reviewer: ```suggestion
Next environment variables are available to customize the build:
```
